docs/angular/api-angular/schematics/karma-project.md

```diff
@@ -6,7 +6,20 @@ Add karma testing to a project
 
 ```bash
 ng generate karma-project ...
+```
+
+By default, Nx will search for `karma-project` in the default collection provisioned in `angular.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+ng g @nrwl/angular:karma-project ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+ng g null ... --dry-run
```

Comment:
There is still null here. This should be ng g karma or ng g @nrwl/karma --dry-run